doc/spinel-protocol-src/spinel-tech-thread.md

```diff
@@ -233,3 +233,24 @@ The leader network data.
 * Packed-Encoding: `D`
 
 The stable leader network data.
+
+### PROP 5391: PROP_THREAD_JOINERS {#prop-thread-joiners}
+
+* Type: Read-Write
+* Packed-Encoding: `A(T(EDc))`
+
+Data per item is:
+
+* `E`: Extended/long address
+* `D`: PSKd
+* `c`: PSKd length
```

Comment:
Alternatively, if `PSKd` is always guaranteed to be a UTF8 string, then you should use `U` instead. Then you don't have to mess with all of this nutty `D.` stuff.